templates/vray/render-linux/job.template.json

```diff
@@ -63,6 +63,12 @@
                 "poolId": "[parameters('poolId')]"
             },
             "jobPreparationTask": {
+                "userIdentity": {
+                    "autoUser": {
+                        "scope": "task",
```

Comment:
change all these to "pool"

Reply:
Actually, I can just remove this userIdentity property. It is unndeed